packages/core/src/app/cart/EditLink.tsx

```diff
@@ -1,22 +1,58 @@
-import React, { FunctionComponent, memo } from 'react';
+import React, { FunctionComponent, memo, useState } from 'react';
 
+import { preventDefault } from '@bigcommerce/checkout/dom-utils';
 import { TranslatedString } from '@bigcommerce/checkout/locale';
+import { ConfirmationModal } from '@bigcommerce/checkout/ui';
 
 export interface EditLinkProps {
     className?: string;
+    isMultiShippingMode: boolean;
     url: string;
 }
 
-const EditLink: FunctionComponent<EditLinkProps> = ({ className, url }) => (
-    <a
-        className={className || 'cart-header-link'}
-        data-test="cart-edit-link"
-        href={url}
-        id="cart-edit-link"
-        target="_top"
-    >
-        <TranslatedString id="cart.edit_cart_action" />
-    </a>
-);
+const EditLink: FunctionComponent<EditLinkProps> = ({ className, url, isMultiShippingMode }) => {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+
+    const gotoCartPage = () => {
+        if (window.top) {
+            window.top.location.href = url;
```

Comment:
Howcome we don't use `window.location.assign()`?

Reply:
I noticed we used `window.top.location` three times in `checkout.tsx`. For example, jumping to the customer registration page.
https://github.com/bigcommerce/checkout-js/blob/366c1fcd6f2bda7e0529f29013468e0d1571c676/packages/core/src/app/checkout/Checkout.tsx#L753

It's good for the embedded checkout, and it behaves similarly to the `assign` method in terms of creating browsing history.

Do you think we need to use `assign` here?

Reply:
I think possibly best to test with `assign`.